### ISSUE TITLE
Cache book covers on the mobile app

### ISSUE THREAD

**Issue by PulsarFTW** [advplyr/audiobookshelf-app#907]
It would be great if cover images could be cached inside the app to increase speed and responsiveness. 

Although this likely low priority.

**Comment by PulsarFTW** [comment]
Sample of perceived slow response:

https://github.com/advplyr/audiobookshelf-app/assets/115924048/a3cc1878-2bad-4c0b-80d7-9242fd35b4a2

**Comment by gh0sti** [comment]
It would be nice if it could do this.

**Comment by KaiStarkk** [comment]
Good idea. As an extension to this, I wonder if it would be possible for clients to cache their own version of the library and metadata entirely, only communicating with the server to check for changes and actually stream / download tracks.
